_posts/2024-9-20-announcing-sprocket

```diff
@@ -0,0 +1,68 @@
+- **Speed**: Sprocket is optimized to run workflows with low overhead, helping users execute their tasks faster and more efficiently.
+- **Community-Driven**: As an open-source project, Sprocket is actively developed and improved by its community, making it adaptable to the needs of users in bioinformatics and beyond.
+
+## VS Code Extension for Sprocket
+
+To streamline the workflow development process, **Sprocket** provides a dedicated **Visual Studio Code extension**. This extension integrates Sprocket directly into the popular code editor, offering a powerful environment for writing, running, and debugging WDL workflows.
+
+Here are some of the features the VS Code extension offers:
+
+- **Syntax Highlighting**: The extension provides WDL syntax highlighting, making your code more readable and easier to navigate.
+- **Auto-completion**: It supports auto-completion for WDL tasks, inputs, and outputs, helping you write WDL scripts faster and with fewer errors.
+- **Integrated Workflow Execution**: You can execute Sprocket workflows directly within VS Code, eliminating the need to switch between the editor and the terminal.
+- **Real-time Error Checking**: Catch issues in your WDL code as you write with integrated real-time error checking.
+- **Output Logs**: View detailed execution logs within the VS Code interface, making debugging faster and more intuitive.
+
+### Getting Started with the VS Code Extension
+
+To begin using the Sprocket VS Code extension, follow these simple steps:
+
+1. **Install the Extension**: Open Visual Studio Code, navigate to the Extensions tab, and search for "Sprocket." Install the extension from the marketplace.
+2. **Configure Sprocket**: Set up the Sprocket executable path by going to your VS Code settings and pointing to the Sprocket installation on your machine.
+3. **Run WDL Workflows**: Open a WDL file in VS Code, and you'll see the Sprocket toolbar to execute and monitor workflows directly from the editor.
+
+For more detailed instructions, you can follow the official [Getting Started Guide for VS Code](https://stjude-rust-labs.github.io/sprocket/vscode/getting-started.html).
+
+
+## Getting Started with Sprocket
+
+Interested in trying out Sprocket for your next project? Head over to the [official Sprocket documentation](https://stjude-rust-labs.github.io/sprocket/overview.html) for detailed installation instructions and usage guides. Whether you are working on local workflows or deploying large-scale computations, Sprocket's flexibility ensures you can adapt it to fit your needs.
+
+
+## What's Next?
+
+Sprocket is under active development, and we welcome contributions from the community! Whether you want to suggest new features, report bugs, or contribute code, check out our [GitHub repository](https://github.com/stjude-rust-labs/sprocket) and join the conversation.
```

Comment:
Perhaps also a call to join the Sprocket channel on Slack?

Reply:
Good point will add that now